.buildkite/scripts/common.sh

```diff
@@ -38,6 +34,30 @@ winlogbeat_changeset=(
   "^winlogbeat/.*"
   )
 
+xpack_libbeat_changeset=(
```

Comment:
I cannot find where this particular changeset is validated. Maybe a typo in the name?

Reply:
Something went wrong in the 3a38f8564ab0ea047d1e2370c265c5e23e2b0977,
let me fix that...